src/MSBuild/app.config

```diff
@@ -39,6 +39,10 @@
           <bindingRedirect oldVersion="0.0.0.0-99.9.9.9" newVersion="15.1.0.0" />
         </dependentAssembly>
         <dependentAssembly>
+          <assemblyIdentity name="System.IO.Compression" culture="neutral" publicKeyToken="b77a5c561934e089" />
+          <bindingRedirect oldVersion="0.0.0.0-99.9.9.9" newVersion="4.1.2.0" />
```

Comment:
I think this should be `oldVersion="0.0.0.0-4.1.2.0"` to match the devenv.exe.config equivalent.